src/main/java/net/dries007/tfc/objects/items/tools/TFCAxeItem.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Work under Copyright. Licensed under the EUPL.
+ * See the project README.md and LICENSE.txt for more information.
+ */
+
+package net.dries007.tfc.objects.items.tools;
+
+import com.google.common.collect.HashMultimap;
+import com.google.common.collect.Multimap;
+import net.minecraft.entity.SharedMonsterAttributes;
+import net.minecraft.entity.ai.attributes.AttributeModifier;
+import net.minecraft.inventory.EquipmentSlotType;
+import net.minecraft.item.AxeItem;
+import net.minecraft.item.IItemTier;
+
+/**
+ * This is needed so we override the damage math done by vanilla
+ *
+ * For comparison:
+ * Vanilla: Tool value (ie: 3.0 for swords) + material damage value (ie: 2.0 for iron) + 1.0 (hand) = 6.0
+ * TFC: Tool value (ie: 1.3 for maces) * material damage value (ie: 5.75 for steel) + 1.0 (hand) ~= 7.5
```

Comment:
And I did that, my thoughts, we can't achieve the same results. It works for vanilla because there are only 4 materials and they all rise linearly. Our approach is to make a new tier worth the upgrade, while also having different kinds of weapons and not only the sword, hence the potential progression (rest assured, it's not far above vanilla, just a bit better).

That said, other reasons:
- To "achieve" the same results, we're gonna have to throw out `IItemTier#getAttackDamage` and pass out the damage value we want directly to the item class (ie: ```SwordItem(tier_ignoring_damage, damage_modifier * TFCItemTier.realDamage), attackSpeed, builder)```. This surely is gonna cause some inconsistency down the line (ie: another mod asks for the `IItemTier#getAttackDamage`).
- `SwordItem` only accepts integer as damage parameter.
Probably there were more, but i can't remember now.

Reply:
I'll buy that.